server/service/src/purchase_order_line/insert/validate.rs

```diff
@@ -32,6 +40,27 @@ pub fn validate(
         return Err(InsertPurchaseOrderLineError::CannotEditPurchaseOrder);
     }
 
+    // check if pack size and item id combination already exists
```

Comment:
Insert and insert from CSV use the same validation function. If duplicate item and pack size combo is inserted - will throw error